docs/api_reference/v1beta3.en.md

```diff
@@ -829,3 +835,67 @@ WeaveNetSpec defines the WeaveNet CNI plugin
 | encrypted | Encrypted | bool | false |
 
 [Back to Group](#v1beta3)
+
+### WebHookAuditLogBatchConfig
+
+
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| bufferSize | BufferSize defines the number of events to buffer before batching. If the rate of incoming events overflows the buffer, events are dropped. | int | false |
+| maxSize | MaxSize defines the maximum number of events in one batch. | int | false |
+| maxWait | MaxWait defines the maximum amount of time to wait before unconditionally batching events in the queue. | metav1.Duration | false |
+| throttle | Throttle defines throttle configuration options. | [WebHookAuditLogThrottleConfig](#webhookauditlogthrottleconfig) | false |
+
+[Back to Group](#v1beta3)
+
+### WebHookAuditLogThrottleConfig
+
+
+
+| Field | Description | Scheme | Required |
+| ----- | ----------- | ------ | -------- |
+| disable | Disable disables webhook throttling. Defaults to false, which corresponds to kube-apiservers default of enabling throttling. | bool | false |
```

Comment:
@kron4eg For a follow up, we have in one struct `disable`, but in other `enable`. I think we should standardize this, we could throw in bool pointers, and use it to properly determine if the value is set or not. Slight inconvenience for us code wise, but makes it easier for users.

Reply:
@kron4eg it's return of the pointers :D 
(for reference Artiom and I had a talk about this exact thing two days ago)